src/onegov/election_day/assets/js/embed.js

```diff
@@ -23,15 +23,28 @@ var appendEmbedCode = function(target, width, height, source, title) {
         .append(
             $('<a>')
             .attr('href', '#')
+            .attr('aria-expanded', false)
+            .attr('id', 'embed-link')
```

Comment:
there are possibly multiple embed codes resulting in duplicate ids "embed-link", see generation of random id a few lines above